app/Listeners/NotificationSentWebhookListener.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Listeners;
+
+use App\Http\Controllers\Backend\WebhookController;
+use App\Models\User;
+use Illuminate\Notifications\DatabaseNotification;
+use Illuminate\Notifications\DatabaseNotificationCollection;
+use Illuminate\Notifications\Events\NotificationSent;
+use Illuminate\Support\Facades\Log;
+
+class NotificationSentWebhookListener
+{
+    public function handle(NotificationSent $event)
```

Comment:
We probably need a check here for the type of Notification that is sent since Laravel uses the system for all sorts of stuff  (such as the verification email and password reset emails).

IMO, ˋ$event->channel === 'database'` should work to filter out unwanted notifications.